testsuite/png/ref/out.txt

```diff
@@ -90,5 +90,18 @@ gimp_gradient:
     Monochrome: No
 smallalpha.png       :    1 x    1, 4 channel, uint8 png
     Pixel (0, 0): 240 108 119 1 (0.94117653 0.42352945 0.4666667 0.003921569)
+cicp:
+  16 x   16, 4 channel, float png
+    channel list: R, G, B, A
+    CICP: 1, 13, 0, 1
+    oiio:ColorSpace: "srgb_rec709_scene"
```

Comment:
Is this correct? Should it have both CICP and oiio:ColorSpace?

Reply:
That all sounds reasonable for now. We know that a more comprehensive color space deduction approach is coming.